_includes/scenarios/integrate/prepare-enmeshed-onboarding-package.md

```diff
@@ -0,0 +1,132 @@
+
+For more details on how to send a Request to create a QR Code containing the reference to a RelationshipTemplate, see the description of the [Get RelationshipTemplate]({% link _docs_use-cases/use-case-transport-get-relationship-template.md %}) usecase.
+{: .notice--info}
+
+<!---{% include rapidoc api_route_regex="^get /api/v2/RelationshipTemplates/{id}$" %}--->
+
+After scanning the QR Code, the App user receives the conditions for establishing a Relationship with your Connector as specified in the RelationshipTemplate. If these are accepted, the App user can now send a Relationship Request to your Connector.
+
+### Onboarding of another Connector
+
+If another Connector wants to send a Relationship Request to your Connector, it must first load a RelationshipTemplate owned by your Connector onto itself. This can be done by sending the Request `POST /api/v2/RelationshipTemplates/Peer` on the other Connector with the following JSON payload: <!--- reference: UkxU... --->
+
+```jsonc
+{
+  "reference": "<your RelationshipTemplate's truncatedReference>"
+}
+```
+
+In doing so, it is necessary to insert the value of the `truncatedReference` property read from the [success response]({% link _docs_integrate/prepare-enmeshed-onboarding-package.md %}#success-response) above into the `reference` property. Alternatively, it is possible to use the following JSON payload specifying the ID and the secret key of the RelationshipTemplate obtained from the same [success response]({% link _docs_integrate/prepare-enmeshed-onboarding-package.md %}#success-response):
+
+```jsonc
+{
+  "id": "<your RelationshipTemplate's ID>",
+  "secretKey": "<your RelationshipTemplate's secret key>"
+}
+```
+
+For more details on how to send a Request to load a RelationshipTemplate created by a Connector onto another Connector, see the description of the [Load RelationshipTemplate created by others]({% link _docs_use-cases/use-case-transport-load-relationship-template-created-by-others.md %}) usecase.
+{: .notice--info}
+
+When the RelationshipTemplate of your Connector is successfully loaded onto the other Connector, a success response is sent. This looks like the above [success response]({% link _docs_integrate/prepare-enmeshed-onboarding-package.md %}#success-response) except that the value of the property `isOwn` is now `false` instead of `true`. Assuming that there is no Relationship between the two Connectors yet and that the RelationshipTemplate contains a data object of type RelationshipTemplateContent in its `content` property, the other Connector will additionally receive a new incoming Request created internally. This can be accepted it if it is wanted to send a Relationship Request to your Connector.
```

Comment:
Can you speficy this request further?
In the following sentence, I'd rewrite the passive construction.

Reply:
This incoming request is explained in detail in another guide. There will be a link to this guide in the "What's next" section when the other guide is published too.

I will rewrite this construction and use the singular "they".